physionet-django/search/templates/search/database_index.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
```

Comment:
Could we have a single template to handle all data types? Seems unnecessary to have separate templates for database and software.